lms/static/js/discovery/models/course_discovery.js

```diff
@@ -24,6 +24,7 @@
             parse: function(response) {
                 var courses = response.results || [];
                 var facets = response.facets || {};
+                //this.courseCards.add(_.uniq(_.pluck(courses, 'data'), x => x.org + "+" + x.number));
```

Comment:
remove this unused code

Reply:
keeping it for now, need to use it later on catalog page